_posts/2020-08-18-react-native-experiences.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Two Years With React Native: Practical Experiences On Android Development"
+author: jarzka
+excerpt: React Native has been around for more than five years now. Is it mature enough for building high quality software while keeping developers happy? Let's find out.
+tags:
+ - React Native
+ - JavaScript
+ - Android
+---
+
+There are many ways to write software for mobile devices. When I started teaching myself mobile application development sometime in 2013, typical options were writing a native application separately for each mobile platform, or by creating a program with web technologies and possibly embedding it into a downloadable "hydrid" application. Both options have their pros and cons, so I decided to learn the essentials of both methods.
+
+React Native entered the field in 2015. It introduced a new "hydrid native" development approach: creating truly native mobile applications by using a single JavaScript codebase which runs on multiple platforms (Android and iOS). Despite of using JavaScript, the big difference compared to embedded web applications is that the user interface layer uses platform specific native components. This makes the user interface of React Native apps feel truly native - because it is! Application logic layer can also use platform specific native APIs via large collection of open source JavaScript libraries. All of this changed my thinking of how mobile applications could be created, because in theory it makes multi-platform mobile application development much easier and faster than before.
+
```

Comment:
hybrid native
which runs -> that runs
Despite of using -> Despite using
platform-specific (X2)
open-source
a large collection
could be created because in theory, (move comma)